heartbeat/monitors/active/icmp/stdloop.go

```diff
@@ -159,29 +158,6 @@ func newICMPLoop() (*stdICMPLoop, error) {
 	return l, nil
 }
 
-func (l *stdICMPLoop) checkNetworkMode(mode string) error {
```

Comment:
Moved to main loop icmp recv so we don't interfere with heartbeat initialization.